plugin.video.dailymotion_com/resources/settings.xml

```diff
@@ -9,9 +9,14 @@
 	<category label="30112">
 		<setting id="downloadDir" type="folder" label="30108" default=""/>
 	</category>
+	<category label="30102">
+		<setting id="forceViewMode" type="bool" label="30102" default="true"/>
```

Comment:
It's okay to allow add a configure setting for forcing the view mode, but it has to be off by default.

Reply:
Thanks for the clarification. Changed and squashed the commit.